core/object/object.h

```diff
@@ -570,10 +574,11 @@ class Object {
 
 	enum ConnectFlags {
 		CONNECT_DEFERRED = 1,
-		CONNECT_PERSIST = 2, // hint for scene to save this connection
+		CONNECT_PERSIST = 2, // Hint for scene to save this connection.
 		CONNECT_ONE_SHOT = 4,
 		CONNECT_REFERENCE_COUNTED = 8,
 		CONNECT_INHERITED = 16, // Used in editor builds.
+		CONNECT_NO_EDITOR = 32, // Used in editor builds.
```

Comment:
I wouldn't understand the name `CONNECT_NO_EDITOR` from a glance. How about something like `CONNECT_SKIP_EMIT_IF_IN_EDITOR`?

Reply:
That sounds too verbose compared to other flags. I'll just add a better comment.

Reply:
I think verbose variable names are better than short variable names with comments :)

Reply:
Yes, but the flag is not that important and can always be renamed in the future anyway.
`CONNECT_SKIP_EMIT_IF_IN_EDITOR` does not properly explain the purpose; needs a better name.